src/Nethermind/Imapp.Benchmark.Runner/BenchmarkNullExporter.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BenchmarkDotNet.Exporters;
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Reports;
+
+namespace Imapp.Benchmark.Runner
+{
+    public class BenchmarkNullExporter : IExporter
+    {
+        public string Name => "ImappNullExporter";
+
+        public IEnumerable<string> ExportToFiles(Summary summary, ILogger consoleLogger)
+        {
+            return Enumerable.Empty<string>();
+
+            // Debugging code
+
+            //Console.WriteLine("Exporter Code: " + Environment.GetEnvironmentVariable("NETH.BENCHMARK.BYTECODE"));
```

Comment:
after we're sure this works, I'll prefer to remove all the commented debugging code, to make this more approachable.

If necessary, leave only some modest logging of executed bytecode to stderr, but maybe best nothing (same as evmone)